gain_tx: 0.01
gain_ty: -0.08
gain_tz: 0.0008
gain_yaw: -0.08